### PR TITLE
Origin/activitylogs dev billing microsoft.billing 2019 10 01 preview

### PR DESCRIPTION
**Pull request by RitikaGeraMicrosoft**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [ check] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [ check] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ check] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [check ] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [x] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

File: specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/billing.json [hunk 1]
```diff
@@ -158,6 +158,145 @@
...
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/billingActivityLogs/{activityLogName}": {
```
Comment (pilor) [inline specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/billing.json:209]: there is no collection GET for /billingActivityLogs that lists all the logs for a billingAccount in a paginated manner.  Any singular GET needs to have a matching collection GET on the resource type

File: specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/examples/BillingActivityEventCategory.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  "parameters": {
+    "api-version": "2019-10-01-preview",
+    "billingAccountName": "{billingAccountName}"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/providers/Microsoft.Billing/billingAccounts/7303440/billingActivityEventCategoryListResult",
```
Comment (pilor) [inline specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/examples/BillingActivityEventCategory.json:11]: these IDs are not appropriate.  To match the API it should be "/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/billingActivityEventCategory/7303440"

File: specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/billing.json [hunk 3]
```diff
@@ -5395,6 +5534,170 @@
...
+          "description": "The properties of an activity event category.",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/BillingActivityEventCategoryProperties"
+        }
+      }
+    },
+    "BillingActivityEventCategoryProperties": {
+      "description": "The properties of an activity event category.",
+      "properties": {
+        "name": {
```
Comment (pilor) [inline specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/billing.json:5677]: name is a top level property on every resource.  It shouldn't be duplicated inside the property bag.

File: specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/billing.json [hunk 4]
```diff
@@ -158,6 +158,145 @@
...
+          },
+          {
+            "$ref": "#/parameters/activityLogNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/BillingActivityRelatedProperty"
```
Comment (pilor) [inline specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/billing.json:236]: the response to this API shoudl look like a resource (with a top level id, name, and type).  This is a GET on a single entity

**Review by pilor — CHANGES_REQUESTED** [review]
Please ensure your APIs conform to the RP contract doc:  https://github.com/Azure/azure-resource-manager-rpc

**Comment by RitikaGeraMicrosoft** [comment]
Abandoning PR

**Pull request closed without merging**